imagery/i.aster.toar/main.c

```diff
@@ -306,14 +303,14 @@ int main(int argc, char *argv[])
                 outrast[i][col] = dout[i];
             }
         }
-        for (i = 1; i <= MAXFILES; i++)
-            Rast_put_row(outfd[i], outrast[i - 1], out_data_type);
+        for (i = 0; i < MAXFILES; i++)
+            Rast_put_row(outfd[i], outrast[i], out_data_type);
     }
-    for (i = 1; i <= MAXFILES; i++) {
-        G_free(inrast[i - 1]);
-        Rast_close(infd[i]);
-        G_free(outrast[i - 1]);
-        Rast_close(outfd[i]);
+    for (i = 0; i < MAXFILES; i++) {
+            G_free(inrast[i]);
```

Comment:
L310-L313: Are these tabs? Why extra whitespaces than before?

Reply:
You're right, will correct that.